src/common-functions.vue

```diff
@@ -184,4 +184,31 @@ export function getOverallRankEmoji(
 
   return null;
 }
+
+export const UtilityCosts = {
+  year: 2021,
+  source: 'https://www.bls.gov/regions/midwest/news-release/AverageEnergyPrices_Chicago.htm',
```

Comment:
This is the actual data source, let's make sure to note this in the data explainer